tools/travis/build.sh

```diff
@@ -37,7 +37,7 @@ export OPENWHISK_HOME=$WHISKDIR
 
```

Comment:
I think build.sh should be updated accrodingly to changes on ansible, in line 31, it should be:
```
$ANSIBLE_CMD wipe.yml
$ANSIBLE_CMD openwhisk.yml
$ANSIBLE_CMD postdeploy.yml
```